net-vpn/i2pd/i2pd-2.54.0.recipe

```diff
@@ -19,12 +19,12 @@ COPYRIGHT="i2pd developers"
 LICENSE="BSD (3-clause)"
 REVISION="2"
 SOURCE_URI="https://github.com/PurpleI2P/i2pd/archive/refs/tags/$portVersion.tar.gz"
-CHECKSUM_SHA256="f5fafa700b61d0791d37bd8eee04912582ea5e3f3b1d80ec339bd8158a30995b"
+CHECKSUM_SHA256="5c3f703417bb5f3e5dda642d39c5d30593a5dcf69d5a5ecfe82d5e8a7d454aaf"
 
 ARCHITECTURES="all !x86_gcc2"
 SECONDARY_ARCHITECTURES="x86"
 
-boostMinumumVersion="1.83.0"
+boostMinumumVersion="1.85.0"
```

Comment:
boost1.85 is broken atm, could this be reverted to use boost1.83?

Reply:
I have no issues on my system, hrev58328

Reply:
That's probably because you still have ICU75 installed :)

Reply:
Can boost 1.85 rebuild with newer ICU? Or where is bug? If it impossible I change boost version as soon as possible

Reply:
ICU75 is currently disabled because it could be problematic to mix multiple ICU versions, if I understood this correctly. The system currently uses ICU74, so that should be used for everything right now.

That means that boost 1.85 should be rebuilt with ICU 74.